packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_flavor_config/__brick__/flavorizr.yaml

```diff
@@ -0,0 +1,104 @@
+app:
+  android:
+    flavorDimensions: "environment"
+
+flavors:
+
+  dev:
```

Comment:
Please use the following flavours

1. development
2. sit
3. uat
4. production